### ISSUE TITLE
Pasting a an odd CR+LF from website crashes Geany (possibly from an odd encoding)

### ISSUE THREAD

**Issue by poikilos** [geany/geany-plugins#1036]
- Geany 1.36
- en_US.UTF-8
- GTK 3.24.23
- GLib 2.64.6

Copying and pasting these four characters crashes Geany 1.36 consistently:
![image](https://user-images.githubusercontent.com/7557867/99807263-40ffb080-2b0d-11eb-9a39-944932d5982d.png)

- Opening the file doesn't crash Geany, but _pasting_ it does.

Steps to reproduce issue
1. Copy from Original source: Mailing address at bottom of <https://videocityinc.com/edit/>
   - optional: Paste into a new text file in `nano` to reproduce issue later (copy from there instead of step 1)
     - copy the whole thing from nano or copy just the 4 characters (using GHex)
2. Paste it into any open text file in Geany.

**Comment by b4n** [comment]
Weird indeed… may be the same as #1022?

**Comment by b4n** [comment]
…or maybe there's a specific spellcheck configuration option that triggers a crash maybe?

**Comment by poikilos** [comment]
> …or maybe there's a specific spellcheck configuration option that triggers a crash maybe?

Correct! I enable "Check spelling while typing" and then it crashes when I paste, even with a temp config. That makes sense.

**Comment by codebrainz** [comment]
> Weird indeed… may be the same as #1022?

Was my first thought too.
